index.js

```diff
@@ -39,6 +40,7 @@ const OUTPUT_MODE = 1;
 const ANALOG_MODE = 2;
 const PWM_MODE = 3;
 const SERVO_MODE = 4;
+const SOFT_PWM_MODE = 82;
```

Comment:
The mode for software PWM should be set to `PWM_MODE`, as this is important for compatibility with Johnny-Five (it won't know what to do with mode `SOFT_PWM_MODE`).